reconciler/benchmark/main.go

```diff
@@ -122,9 +122,9 @@ func main() {
 			"test",
 			"Test",
 
```

Comment:
commit msg typo:

`With Hive cannot currently have to private providers for the same type,
even when they're in different scopes.`

should probably be `With Hive _you_ cannot` and then `have _two_ private`

Reply:
Thanks!